scheduler/pkg/envoy/xdscache/resource.go

```diff
@@ -64,6 +65,7 @@ func makeHTTPListener(listenerName, address string,
 	port uint32,
 	routeConfigurationName string,
 	serverSecret *Secret,
+	config *EnvoyConfig,
```

Comment:
+1 I quite like how this opens the possibility of us surfacing lots of other envoy settings as well